app/controllers/api/v1/ecf_users_controller.rb

```diff
@@ -48,7 +48,7 @@ def email
       end
 
       def users
-        Api::V1::ECF::UsersQuery.new(updated_since:, email:).all
```

Comment:
could we also remove the rake task that diffs those out as those classes no longer exist?

Reply:
I've removed the rake task, thanks for the spot.

On the query about "other" records. I was being consistent with the original implementation, however if you look at the actual code for ECT Support it just ignores these records.

There is more work that needs to be done to ECT Support and this API, but I wanted to do it in small steps rather than everything in one massive bang especially as there is no clear direction on how we want to handle things like multi role access and access for SITs or others. I think that this is a really poorly thought through API to be honest, its neither completely agnostic of usage or completely controlling of function so the actual logic for who should have access is shared across both applications.

If the POs want me to progress with sorting these service problems out I will, but I am not sure they want me to without proper research and design support.